cookbooks/rubygems-logging/templates/default/logstash/output_elasticsearch.conf.erb

```diff
@@ -1,5 +1,6 @@
 output {
   elasticsearch {
-    host => "127.0.0.1"
+    host => [ "127.0.0.1" ]
+    protocol => "http"
```

Comment:
Default protocol is "node". When logstash acts as node, it doesn't store the data, but queries (kibana or else) are still routed to it, which makes no sense and leads to couple of issues (there are upstream tickets related to that). Setting to HTTP as the most universal protocol.